### DISCUSSION TITLE
Few asks on Mighty Mouse support [other]

### DISCUSSION THREAD

**Discussion by etienner76** [noah-nuebling/mac-mouse-fix#189] [Questions] [answered]
Hi,

Starting this off by wishing you the best for 2022!

Few asks on Mighty Mouse support:
- Is the wireless Mighty Mouse supported by Mac Mouse Fix?
- If so, how do I make use of the middle button function?
- Any way to have an option to boost tracking speed? It is notoriously slow, even maxing out the default settings
- Is Mac Mouse Fix enable even though I close the App?

Cheers,

-Etienne

**Comment by noah-nuebling** [comment, marked as answer]
Hey Etienne, thanks for your feedback, I hope you have a nice 2022, too! :)

I don't have a Mighty Mouse to test, so I don't know if it's supported. But you can try the [2.0.0 Beta](https://github.com/noah-nuebling/mac-mouse-fix/releases), and see if that works (click the "+" button to add actions to your buttons).

Mac Mouse Fix doesn't have feature to boost tracking speed as of yet, but [LinearMouse](https://linearmouse.org/) should work for that.

Mac Mouse Fix stays enabled after you close the app.

Hope that helps and let me know if you have any more questions!

Noah

**Reply by noah-nuebling** [reply]
I've moved the topic of Mighty Mouse Support to Issue #203. So just refer to that if you have more questions. Thanks!

Marking this as answered to clean things up.
